portfolio/src/main/webapp/index.html

```diff
@@ -30,8 +30,11 @@ <h1 class="attention">Chiemeziem Nwaiwu.</h1>
             </div>
             <img id="my-pic" src="images/Chiemeziem.jpg" class="" alt="My picture" title="It's Chiemeziem!" />
         </div>
-        <button type="button" class="button" onclick="addFunFacts()">Guess What!</button>
+        <button type="button" class="button" onclick="addFunFacts()">Random Facts About ME!</button>
         <div id="facts-container"></div>
+        <button type="button" class="button" onclick="getFacts()">Another Fact</button>
+        <div id="quote-container"></div>
+        <!-- <p>Click <a href="/date">here</a> to see the server response directly.</p> -->
```

Comment:
While commenting out code is useful while you're working. When committing to a shared project, it should generally be removed. In this case, you can get it back through git history if you need it.

Reply:
I thought I was eventually going to use the snippet but I did not. I will be removing it in the next PR.